spec/controllers/admin/companies_controller_spec.rb

```diff
@@ -5,6 +5,9 @@
 
   before(:each) do
     @company = create(:company)
+    admin = User.create(email: "admin@example.com", password: "password", role: 1)
+    allow_any_instance_of(ApplicationController).to receive(:current_user)
+      .and_return(admin)
   end
```

Comment:
We had to stub out the current user as an admin in order to test these controller actions because of the authorization that we added on the admin companies controller. FYI